modules/sand/src/main/ts/ephox/sand/core/OperatingSystem.ts

```diff
@@ -7,7 +7,7 @@ export interface OperatingSystem extends UaInfo {
   readonly isWindows: () => boolean;
   readonly isiOS: () => boolean;
   readonly isAndroid: () => boolean;
-  readonly isOSX: () => boolean;
+  readonly isMacOS: () => boolean;
```

Comment:
This will also need a changelog entry for sand

- [ ] Add sand changelog entry

Reply:
You've also missed some cases here looking more. In this file you'll see this which also needs updating: https://github.com/tinymce/tinymce/blob/458d5a4a39b2f160d0b48d89219883407a27375e/modules/sand/src/main/ts/ephox/sand/core/OperatingSystem.ts#L21

- [ ] Ensure the name is also updated to `macOS`